charts/ocean-network-client/templates/configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
@yehielnetapp 
We must include 

disable-auto-update: {{ default "false" .Values.spotinst.disableAutoUpdate | quote }}